roles/ansible/tower/manage-schedules/templates/schedule.j2

```diff
@@ -1,16 +1,28 @@
 {
     "rrule": "{{ schedule.rrule }}",
     "name": "{{ schedule.name }}",
-    "description": "{{ schedule.description | default('') }}",
+{% if schedule.description is defined %}
+    "description": "{{ schedule.description }}",
+{% endif %}
+{% if schedule.extra_data is defined %}
     "extra_data": {{ schedule.extra_data | default("{}") | to_json }},
```

Comment:
probably just remove the `default` part of all of these that now have conditional checks(?)